### PR TITLE
When enrolling teachers in plc courses that have already started, hav…

### PR DESCRIPTION
**Pull request by mehalshah**
…e their enrollments start as IN PROGRESS instead of START BLOCKED

### CODE REVIEW

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -425,6 +425,7 @@
     t.datetime "created_at",                   null: false
     t.datetime "updated_at",                   null: false
     t.integer  "script_id",        limit: 4
+    t.boolean  "started"
```
Comment (aoby) [inline dashboard/db/schema.rb]: should this have `default: false, null: false` or is there value in allowing null?

File: dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AddStartedToPlcCourseUnit < ActiveRecord::Migration
+  def change
+    add_column :plc_course_units, :started, :boolean
```
Comment (aoby) [inline dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb]: should this have `default: false, null: false` or is there value in allowing null?

File: dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+    Plc::CourseUnit.update_all(started: true)
```
Comment (aoby) [inline dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb]: all existing course units are started?

File: dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb (hunk 2 above)
Comment (mehalshah) [inline dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb]: I like that.

File: dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb (hunk 3 above)
Comment (mehalshah) [inline dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb]: Yes, as of right now. Although now that I think about it, that might not be the case when this gets checked in. I'm fine removing it and doing this manually

**Comment by aoby** [comment]
lgtm

**Pull request merged**